README.md

```diff
@@ -21,13 +21,13 @@ Two reasons:
 
 ## Should I uninstall linter-eslint?
 
-Depends. The `linter-eslint` package supports **ESLint up through and including v7**. This new package supports **ESLint v7 and greater**. The overlap in v7 is because that’s the one major version where both interfaces, `CLIEngine` and `ESLint`, are available.
+Depends. The `linter-eslint` package supports **ESLint up through and including v7**. This new package supports **ESLint v8 and greater**.
 
-If _all_ your projects use ESLint `>=7.0.0`, you can keep this package and uninstall `linter-eslint`. If _any_ of your projects use an older ESLint, you should keep `linter-eslint` installed alongside this package. **This package can coexist with `linter-eslint`; they won’t get in each other’s way.**
+If _all_ your projects use ESLint `>=8.0.0`, you can keep this package and uninstall `linter-eslint`. If _any_ of your projects use an older ESLint, you should keep `linter-eslint` installed alongside this package.
 
-Since they can both lint when ESLint 7.x is present, they have to coordinate who does the linting when both packages are installed. **If `linter-eslint` is installed, this package will not perform linting in ESLint 7.x environments** — only 8.x or greater. If only this package is installed, it will lint with any version of ESLint it supports.
+If only this package is installed, it will lint with any version of ESLint it supports.
 
-When `linter-eslint` is not installed and this package detects an ESLint version too old for it to support, it will show a notification and invite you to install `linter-eslint`. This behavior can be disabled in package settings.
+When `linter-eslint` is not installed and this package detects an ESLint version too old for it to support, it will show a notification and invite you to install `linter-eslint`. This behaviour can be disabled in package settings.
```

Comment:
Oh, man, I just noticed this change. You trying to pick a fight? 😆

Reply:
Here was me just removing the little red lines in my editor! 😂 🥊 😱

I can change it back 👀

Reply:
Nah, I'll get it at some point, no worries